brand/models/state.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+COUNTRIES = {"US": "United States", "CA": "Canada", "AU": "Australia"}
+
+
+class State(models.Model):
+    tag = models.CharField(max_length=100, primary_key=True, null=False, blank=False)
+    name = models.CharField(max_length=100, null=False, blank=False)
+
+    """
+    Country code of Bank.Green supported country states
+    Must be compatable with django_countries `code` (ISO 3166-1 alpha-2)
+    """
+
+    class CountryCode(models.TextChoices):
```

Comment:
I'm surprised you didn't go for Django countries here. It's not a major issue though so I'm happy with this piece as is